packages/cardano-services-client/src/TxSubmitProvider/TxSubmitApiProvider.ts

```diff
@@ -0,0 +1,47 @@
+import { Cardano, ProviderError, ProviderFailure, SubmitTxArgs, TxBodyCBOR, TxSubmitProvider } from '@cardano-sdk/core';
+import { Logger } from 'ts-log';
+import { hexStringToBuffer } from '@cardano-sdk/util';
+import axios, { AxiosInstance } from 'axios';
+
+export class TxSubmitApiProvider implements TxSubmitProvider {
+  #axios: AxiosInstance;
+  #logger: Logger;
+  #path: string;
+
+  constructor(submitApiUrl: URL, logger: Logger) {
+    this.#axios = axios.create({ baseURL: submitApiUrl.origin });
+    this.#logger = logger;
+    this.#path = submitApiUrl.pathname;
+  }
+
+  async submitTx({ signedTransaction }: SubmitTxArgs) {
+    let txId = '' as Cardano.TransactionId;
+
+    try {
+      txId = Cardano.TransactionId.fromTxBodyCbor(TxBodyCBOR.fromTxCBOR(signedTransaction));
+
+      this.#logger.debug(`Submitting tx ${txId} ...`);
+
+      await this.#axios({
+        data: hexStringToBuffer(signedTransaction),
+        headers: { 'Content-Type': 'application/cbor' },
+        method: 'post',
+        url: this.#path
+      });
+
+      this.#logger.debug(`Tx ${txId} submitted`);
+    } catch (error) {
+      this.#logger.error(`Tx ${txId} submission error`);
+      this.#logger.error(error);
+
+      // TODO improve (implement?) error handling
+      throw axios.isAxiosError(error)
+        ? new ProviderError(ProviderFailure.BadRequest, null, error.response?.data as string)
+        : new ProviderError(ProviderFailure.Unknown, error, 'submitting tx');
+    }
+  }
+
+  healthCheck() {
+    return Promise.resolve({ ok: true });
```

Comment:
I was going to suggest calling a path we know not to exist, and assert a 404, but it's more important to ship something that works ASAP. Maybe we should rethink if this method is even needed 🤔

Reply:
> I was going to suggest calling a path we know not to exist, and assert a 404
I thought the same, but I wasn't able to find a way to perform an ok dry call to get the `ok` status.

Reply:
An improvement over what we currently have is mapping status of last submission: `2xx` or `4xx` is ok, everything else is not ok

Reply:
I don't think it's meaningless: afaik client is not checking health before submitting, but health is used to track backend services

Reply:
b6dc40411 should solve this